<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_einstellung_dxf_klassen</class>
 <widget class="QDialog" name="Dialog_einstellung_dxf_klassen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Klasseneinstellungen DXF</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Bohrung horizontal</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_fraes_vert"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>8,5mm breit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_wst_beispiel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>14,2mm tief</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_nut_vert"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Nut vertikal</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>19,6mm Dick</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_wst"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Bauteil</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="lineEdit_nut_vert_beispiel"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_bohr_hori"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Klassenname</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>14,2mm tief</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_bohr_vert"/>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="lineEdit_bohr_hori_beispiel"/>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>durchgefräst 2mm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Anwendungsbeispiel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Fräser vertikal</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="lineEdit_fraes_vert_beispiel1"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Bohrung vertikal</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_bohr_vert_beispiel"/>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="lineEdit_fraes_vert_beispiel2"/>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>5mm tief eingefräst</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>(Beisp. jew. mit Werkzeug 601 mit Radiuskorrektur links)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_speichern">
      <property name="text">
       <string>Speichern</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
